README.md

```diff
@@ -1 +1,101 @@
-# capstone-project-kanban
\ No newline at end of file
+# Project Name
+
+capstone-project-kanban
+
+[]: # Language: markdown
+[]: # Path: README.md
+
+# Description the project.
+
+This is the second Microverse capstone project meant to summarize all the content learned through module 2. The objective was to mimic the appearance of a dynamic web application, in a similar way a web developer would receive the requirements as if it were in a job.
+
+The pull request is opened to get a review on the project and check for any required changes.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+- Gitflow
+- Jest testing
+- Kanban Board (github project)
+- following github flows standards.
+
+## Getting Started
+
+**This web page is books site with my List planned or shores to do in the future**
+
+**To do List project works on mobile and desktop using @Media queries .**
+
+---
+
+To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
+
+### Setup
+
+to use the project To do list please follow :
+
+> Open GitHub and go to my GitHub repository name: capstone-project-kanban.to clone.
+> Click “Code” and copy the given URL.
+> click code and copy the URL.
+
+> click code and copy the URL.
+
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+> Syntax:
+
+$ git clone {repository URL}
+
+### Usage
+
+> open your directory capstone-kanban/
+
+\_use vs-code or any IDE to view line code .
+
+---
+
+$ npm start to run the local server
+
+---
+
+## Click here for live version
+
+- live demo: [live demo](https://nickemma.github.io/capstone-project-kanban/)
+
+## Authors
+
+👤 **Nicholas Emmanuel**
+
+- GitHub: [@githubhandle](https://github.com/NickEmma)
+- Twitter: [@twitterhandle](https://twitter.com/techieEmma)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/nicholas-emmanuel-6b9775207)
+
+👤 **ADEBOWALE ADEGBOYE**
+
+- GitHub: [@githubhandle](https://github.com/ademibowale)
+- Twitter: [@twitterhandle](https://twitter.com/Ademibowale1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/adebowale-adegboye-143568221/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+thanks:
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [REQUIRED] You should record a video presenting the features of the project and paste the link of your presentation video to your `README` file.  The maximum length of the video is 5 minutes.

![image](https://user-images.githubusercontent.com/63915024/181603454-33f73b2f-1f81-452a-bf39-c0a18754b26c.png)

Reply:
The zoom video recording is on the pull request